modules/widgets/social-media-icons.php

```diff
@@ -74,15 +74,7 @@ public function widget( $args, $instance ) {
 		$html .= '<ul>';
 
 		$alt_text = esc_attr__( 'View %1$s&#8217;s profile on %2$s', 'jetpack' );
-
-		/**
-		 * Fires in the beginning of the list of Social Media accounts, inside the unordered list.
-		 *
-		 * Can be used to add a new Social Media Site to the Social Media Icons Widget.
-		 *
-		 * @since 3.7.0
-		 */
-		do_action( 'jetpack_social_media_icons_widget_list_before' );
```

Comment:
Sorry, my bad, I re-did it after reviewing another pull request and noting that it needed a way to differentiate between numbered pages and custom G+ Urls. I added that and accidentally removed some other stuff... I will re-do and resubmit!